scripts/deploy/govcms-pre-deploy

```diff
@@ -1,26 +1,25 @@
 #!/usr/bin/env bash
-IFS=$'\n'
+IFS=$'\n\t'
```

Comment:
here and below - using consistent `IFS` across scripts - unless this was inconsistent intentionally?

Reply:
Not intentional. looks good

Reply:
That was intentional for this script, it allows us to add multiple commands via the env var separated by a space.

Reply:
ok great.. possibly should comment or modify it when it needs to be modified? Setting a default is to ensure unexpected events.